helm/v1beta1/types_project_helm_chart_repository.go

```diff
@@ -19,13 +19,60 @@ type ProjectHelmChartRepository struct {
 	// spec holds user settable values for configuration
 	// +kubebuilder:validation:Required
 	// +required
-	Spec HelmChartRepositorySpec `json:"spec"`
+	Spec ProjectHelmChartRepositorySpec `json:"spec"`
 
 	// Observed status of the repository within the namespace..
 	// +optional
 	Status HelmChartRepositoryStatus `json:"status"`
 }
 
+// Project Helm chart repository exposed within a namespace
+type ProjectHelmChartRepositorySpec struct {
+
+	// If set to true, disable the repo usage in the cluster/namespace
```

Comment:
this is specifically the namespace now, right?  and the other is specifically the cluster.

Reply:
Yes this will be specifically namespace now. Will update that.